### ISSUE TITLE
Missing options keys

### ISSUE THREAD

**Issue by demurgos** [rogierschouten/gulp-typedoc#27]
The type definitions are missing some of the keys accepted by Typedoc. For example, I get errors when setting `tsconfig` or `ignoreCompilerErrors`.
Since the options are mostly passed as-is to `typedoc`, it would be better if the options of this lib were derived from those of `typedoc`.

**Comment by rogierschouten** [comment]
Yes you're completely right. When I wrote this, the typescript primitives for manipulating types weren't yet good enough. 

Any chance of a PR? Otherwise, this might take a while as I'm quite busy atm.  I do promise to merge & publish a PR quickly.

**Comment by rogierschouten** [comment]
Btw it might even be a better idea to just rewrite this in Typescript altogether, it's just a couple of lines of code... whatever we can manage.

**Comment by demurgos** [comment]
I am in a period when I am updating various build tools, so I'll try to send a PR. Would it be OK if I converted the code to TS first, so the `.d.ts` file could be generated?

**Comment by demurgos** [comment]
Okay, I see that we posted comments at the same time, so it answers my question :smile:
